src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -437,4 +437,7 @@
   <data name="DelegateInvocationCanBeSimplified" xml:space="preserve">
     <value>Delegate invocation can be simplified.</value>
   </data>
-</root>
+  <data name="AddSwitchLabels" xml:space="preserve">
+    <value>Add Missing Labels</value>
```

Comment:
Text should be "Add missing switch cases" or "Add missing switch labels".  the data name should then match that text.  Right now these guys are out of sync.